### ISSUE TITLE
baichuan2 13b 多卡pt 进程卡死。单卡可以正常进行

### ISSUE THREAD

**Issue by sunshineyg2018** [shibing624/MedicalGPT#218]
### Describe the bug
Please provide a clear and concise description of what the bug is. If applicable, add screenshots to help explain your problem, especially for visualization related problems.

参数：
CUDA_VISIBLE_DEVICES=0,1,2,3 torchrun --nproc_per_node 4 pretraining.py \
    --model_type baichuan \
    --model_name_or_path /root/autodl-tmp/baichuan2_inc_13 \
    --train_file_dir /root/train_code/MedicalGPT/data/pretrain \
    --validation_file_dir /root/train_code/MedicalGPT/data/pretrain \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --do_train \
    --do_eval \
    --use_peft True \
    --seed 42 \
    --fp16 \
    --max_train_samples -1 \
    --max_eval_samples -1 \
    --num_train_epochs 0.5 \
    --learning_rate 2e-4 \
    --warmup_ratio 0.05 \
    --weight_decay 0.01 \
    --logging_strategy steps \
    --logging_steps 10 \
    --eval_steps 50 \
    --evaluation_strategy steps \
    --save_steps 500 \
    --save_strategy steps \
    --save_total_limit 3 \
    --gradient_accumulation_steps 1 \
    --preprocessing_num_workers 1 \
    --block_size 1024 \
    --output_dir /root/autodl-tmp/outputs_pt_baichuan2_13_v1 \
    --overwrite_output_dir \
    --ddp_timeout 30000 \
    --logging_first_step True \
    --target_modules all \
    --lora_rank 8 \
    --lora_alpha 16 \
    --lora_dropout 0.05 \
    --torch_dtype float16 \
    --device_map auto \
    --report_to tensorboard \
    --ddp_find_unused_parameters False \
    --gradient_checkpointing True \
    --cache_dir /root/autodl-tmp/cache

**Comment by sunshineyg2018** [comment]
单卡正常

**Comment by shibing624** [comment]
显存不足？

**Comment by sunshineyg2018** [comment]
> 存量明显不足？

是指内存吗
